babi/file.py

```diff
@@ -673,39 +673,53 @@ def sort_selection(self, margin: Margin, reverse: bool = False) -> None:
     def _is_commented(self, lineno: int, prefix: str) -> bool:
         return self.buf[lineno].lstrip().startswith(prefix)
 
-    def _comment_remove(self, lineno: int, prefix: str) -> None:
-        line = self.buf[lineno]
+    def _start_space_offset(self, line: str) -> Tuple[int, str]:
```

Comment:
I think the code is simpler if you don't return the string here -- looks like a bit of over-DRYing especially since most callers discard the string

Reply:
You right, I cleaned it.